src/main/java/tw/edu/ntu/lowerbound10hours/jlaskhibernate/Hibernate.java

```diff
@@ -0,0 +1,70 @@
+package tw.edu.ntu.lowerbound10hours.jlaskhibernate;
+
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.cfg.Configuration;
+import tw.edu.ntu.lowerbound10hours.jlask.Jlask;
+
+public class Hibernate {
+  private SessionFactory factory = null;
+  private Session session = null;
+  private Transaction tx = null;
+  private Configuration config = null;
+  private Jlask app = null;
+
+  public Hibernate() {
+    this.config = new Configuration().configure();
+  }
+
+  public Hibernate(Jlask app) {
+    this.app = app;
+    this.config = new Configuration().configure();
+  }
+
+  /** Get the session from the sessionFactory. */
+  public Session create_scoped_session() {
+    this.create_session();
+    Session session = this.factory.openSession();
+    return session;
+  }
+
+  /** Create the sessionFactory based on the configuration. */
+  public void create_session() {
+    try {
+      this.factory = this.config.buildSessionFactory();
+    } catch (Throwable ex) {
+      System.err.println("Failed to create sessionFactory object." + ex);
+      throw new ExceptionInInitializerError(ex);
+    }
+  }
+
+  /** Initialize the Hibernate object. */
+  public void init_app(Jlask app) {
```

Comment:
Please use camelCase, so as the following ones

Reply:
Updated